apps/ledger-live-desktop/src/renderer/screens/settings/sections/Accounts/CurrencyRows.tsx

```diff
@@ -62,13 +62,19 @@ function CurrencyRows({ currency }: Props) {
         >
           <Track onUpdate event="ConfirmationsNb" confirmationsNb={confirmationsNb} />
           {defaults.confirmationsNb ? (
-            <StepperNumber
-              min={defaults.confirmationsNb.min}
-              max={defaults.confirmationsNb.max}
-              step={1}
-              onChange={handleChangeConfirmationsNb}
-              value={confirmationsNb}
-            />
+            <Box
+              style={{
+                width: 150,
+              }}
+            >
```

Comment:
I think our `Box` should have props for styles
```suggestion
            <Box width={150}>
```